interfaces/console/src/components/NodeStatItem/index.tsx

```diff
@@ -88,14 +89,14 @@ const NodeStatItem = ({
 }: INodeStatItem) => {
   return (
     <Grid container spacing={2}>
-      <Grid item xs={variants(variant, 'NG')}>
+      <Grid size={{ xs: variants(variant, 'NG') }}>
```

Comment:
Why do you use `size` instead of `xs`? I believe `size` is deprecated.

Reply:
No its other away around. Grid is deprecated we need to gradually update Grid to Grid v2
https://mui.com/material-ui/migration/migration-grid-v2/